locality-checker/main.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"context"
+	"log"
+	"os"
+
+	"github.com/cockroachdb/k8s/locality-checker/pkg/kubernetes"
+)
+
+const localityMountPath = "/etc/cockroach-locality"
```

Comment:
Consideration for future proofing: Accept as `os.Args[1]` but default to this value?